index.html

```diff
@@ -61,5 +70,166 @@ <h1>Hey There.<br>I'm Daniel</h1>
             </div>
         </div>
     </section>
+
+    <section class="work-section">
+        <div class="container">
+        <div class="title">
+            <h1><span>My Recent work</span></h1>
```

Comment:
- [ ] It is a good coding practice to have only one `h1` tag for the entire web page. This helps in page indexing and SEO. I therefore suggest that you choose a different heading tag to use in replacement. The `h1` tag was already used in the upper sections.